project.clj

```diff
@@ -37,8 +37,7 @@
                 *print-length* 100}
   ;;FIXME: https://github.com/technomancy/leiningen/issues/2173
   :monkeypatch-clojure-test false
-  :jvm-opts ["-Xmx1g" "-XX:-OmitStackTraceInFastThrow"
-             "-XX:+UseG1GC" "-XX:+UseStringDeduplication"]
+  :jvm-opts ["-Xmx1g" "-XX:-OmitStackTraceInFastThrow"]
```

Comment:
And why those are useless here? ;)

Reply:
no need for a big garbage collector in development ;)